amundsen_application/proxy/issue_tracker_clients/jira_client.py

```diff
@@ -67,12 +70,15 @@ def create_issue(self, table_uri: str, title: str, description: str) -> DataIssu
         :return: Metadata about the newly created issue
         """
         try:
+            user_email = app.config['AUTH_USER_METHOD'](app).email
```

Comment:
should you check for null/empty before running split?

Reply:
For this one I'm going to check for `app.config['AUTH_USER_METHOD']`. This method is typed as returning a `User` and that type guarantees an `email` attribute.